### PR TITLE
Update Area name in text and percentages being 0

### PR DESCRIPTION
**Pull request by Kieken72**

### CODE REVIEW

**Comment by Kieken72** [comment]
#159 Fixed with this

File: gyms.py [hunk 1]
```diff
@@ -76,6 +79,8 @@ def get_stats():
                 reverse=True
             )[0]
             top_guardians[team.value] = POKEMON_NAMES[pokemon_id]
+
+    print percentages
```
Comment (Aiyubi) [inline gyms.py]: the print is a leftover I guess?

Comment (Kieken72) [inline gyms.py]: Yeah sorry, removed it!

File: gyms.py [hunk 2]
```diff
@@ -8,6 +8,7 @@
 import config
 import db
 import utils
+from decimal import *
```
Comment (modrzew) [inline gyms.py]: Ugh! Never ever `import *`! More so if you're only going to use `Decimal` from that module :crying_cat_face:

**Comment by modrzew** [comment]
Um... so what exactly does it fix?

**Comment by Kieken72** [comment]
@modrzew In the GYM report, the percentages would show as 0.0 and there was a hardcoded area name in the HTML.

File: gyms.py (hunk 2 above)
Comment (Kieken72) [inline gyms.py]: Sorry, I'm not used to python. I was justing looking for a quick fix to these 0.0%. What should I do then?

**Comment by modrzew** [comment]
Oh, right. There's simpler way: you can just convert one of the values to float to force float division.

```py
count.get(team.value) / float(len(forts)) * 100
```

Or... you can use Python 3!

**Comment by Kieken72** [comment]
Yeah I was just reading it that I should upgrade to Python 3! Thanks

**Comment by modrzew** [comment]
Did it with oldschool `float(...)` in 14cf46935547cfdac84da6ca802bca90957bf1d4.

**Pull request closed without merging**